src/docs/design/anatomy.mdx

```diff
@@ -5,45 +5,39 @@ import PatternStatesImg from '../../app/assets/resource-anatomy/pattern-states.p
 
 # Design System Anatomy
 
-The building blocks of the PX Blue design system are broken down into five levels of increasing complexity: elements, components, templates, pages, and workflows. When communicating with other members of the PX Blue community, you should use strive to use this terminology to promote effective collaboration and understanding. This is applicable to designers, developers, product managers, etc.
+The building blocks of the PX Blue design system are broken down into four levels of increasing complexity: elements, components, pages, and workflows. When communicating with other members of the PX Blue community, you should strive to use this terminology to promote effective collaboration and understanding. This is applicable to designers, developers, product managers, etc.
 
 <ImageGrid
     images={[CategoriesImg]}
     gridImageProps={{ xs: 12, sm: 12, md: 12 }}
-    caption={'The five levels of the PX Blue design system hierarchy.'}
+    caption={'The four levels of the PX Blue design system hierarchy.'}
 />
 
-## Elements
+## Foundations
 
-Elements are the most basic visual elements on the page, such as [color](/style/color), [icons](/style/iconography), [typography](/style/typography), etc. These typically have no behavior or interaction on their own, but instead are used to communicate information to a user.
+The foundations include the most basic visual elements on the page, such as [color](/style/color), [icons](/style/iconography), [typography](/style/typography), etc. These typically have no behavior or interaction on their own, but instead are used to communicate information to a user, theme our brand, or lay the foundations for the PX Blue components, pages, and workflows.
```

Comment:
"Theme our brand" is a strange wording, can we rephrase this? Maybe "establish a brand"